development/compiling/compiling_for_android.rst

```diff
@@ -8,147 +8,154 @@ Compiling for Android
 Android NDK.
 
-To set those environment variables on Windows, press Windows+R, type
+To set those environment variables on Windows, press **Windows + R**, type
 "control system", then click on **Advanced system settings** in the left
 pane, then click on **Environment variables** on the window that
 appears.
 
-To set those environment variables on Unix (e.g. Linux, macOS), use
+To set those environment variables on Linux or macOS, use
 ``export ANDROID_HOME=/path/to/android-sdk`` and
-``export ANDROID_NDK_ROOT=/path/to/android-ndk``.
-Where /path/to/android-sdk and /path/to/android-ndk is the path where Android SDK
-and Android NDK are placed on your PC.
+``export ANDROID_NDK_ROOT=/path/to/android-ndk``
+where ``/path/to/android-sdk`` and ``/path/to/android-ndk`` point to
+the root of the SDK and NDK directories.
 
 Building the export templates
 -----------------------------
 
 Godot needs two export templates for Android: the optimized "release"
-template (`android_release.apk`) and the debug version (`android_debug.apk`).
-Compiling the standard export templates is done by calling scons with
+template (``android_release.apk``) and the debug template (``android_debug.apk``).
+Compiling the standard export templates is done by calling SCons with
 the following arguments:
 
--  Release template (used when exporting with "Debugging Enabled" OFF)
+-  Release template (used when exporting with **Debugging Enabled** unchecked)
 
 ::
 
-    C:\godot> scons platform=android target=release
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
+    scons platform=android target=release
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux
+    ./gradlew build
 
-(on Linux or macOS, execute the `gradlew` script with `./gradlew build`)
 
-The resulting APK is in:
-
-::
+The resulting APK will be located at ``bin/android_release.apk``.
 
-    bin\android_release.apk
-
--  Debug template (used when exporting with "Debugging Enabled" ON)
+-  Debug template (used when exporting with **Debugging Enabled** checked)
 
 ::
 
-    C:\godot> scons platform=android target=release_debug
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
-
-The resulting APK is in:
+    scons platform=android target=release_debug
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
-::
 
-    bin\android_debug.apk
+The resulting APK will be located at ``bin/android_debug.apk``.
 
 Faster compilation
 ~~~~~~~~~~~~~~~~~~
 
-If you are on Unix or installed PyWin32 on Windows and have multiple CPU
-cores available, you can speed up the compilation by adding the ``-jX``
-argument to the SCons command, where ``X`` is the number of cores that you
-want to allocate to the compilation, e.g. ``scons -j4``.
-
+If you have multiple CPU cores available, you can speed up compilation
```

Comment:
That section can be removed, it's already covered in the intro to the buildsystem linked above in the `.. seealso` section.